index.html

```diff
@@ -484,6 +490,115 @@
               layout="[[layoutClass]]"
               route="{{subRoute}}"></generic-page>
           </template>
+          <!-- LURES    ============================================   -->
```

Comment:
Also.  There currently are no links to those lure pages.   This grabs all the lure info in .md files, which is key to easily putting them in place in the near future.